src/dashboard/TasksListFilter.tsx

```diff
@@ -29,10 +29,10 @@ export const TasksListFilter = ({
             sort: { field: 'due_date', order: 'ASC' },
             filter: {
                 ...filter,
-                'contact_id@in': `(${contacts.map(contact => contact.id)})`,
+                'sales_id@in': `(${identity?.id})`,
```

Comment:
why keep the `in` operator here? :thinking: 

```suggestion
                sales_id: identity?.id,
```